docs/getting_started/execution-modes.rst

```diff
@@ -209,6 +214,27 @@ Each task will create a new container on Azure, giving full isolation. This, how
         },
     )
 
+GCP Cloud Run Job
```

Comment:
Two recommendations on the docs:
1. Add a link to Cloud Run documentation
2. Mention that for this execution mode to work, users should, previously:
* Create container
* Create Cloud Run job
3. (optional) If we could have something similar to this, but for GCP Cloud Run, it would be amazing:
https://github.com/astronomer/astronomer-cosmos/blob/main/docs/getting_started/azure-container-instance.rst

Reply:
1 and 2 updated, working on 3.

Reply:
Hello @tatiana,

I created step-by-step guide to setup GCP infrastructure and Airflow to validate the feature: docs/getting_started/gcp-cloud-run-job.rst

All of the files needed for tests are available in cosmos-example repo, [PR #8](https://github.com/astronomer/cosmos-example/pull/8).

By the way, I believe there is still another [PR #6](https://github.com/astronomer/cosmos-example/pull/6) pending on that repo.

Reply:
Thank you very much, @ags-de , this looks great!
Thanks for pointing out the lack of reviews in the cosmos-example repo. I created #1222 so we can address this in the mid-long term